routers/user.js

```diff
@@ -2,13 +2,13 @@ const { Router } = require('express');
 const userRouter = Router();
 const {
   getUserByName,
-  createUser,
+  addUser,
   putBooksInCart,
 } = require('../controllers/user');
 const { userValidation } = require('../middlewares');
 
 userRouter.get('/find', getUserByName);
-userRouter.post('/create', createUser);
+userRouter.post('/add', addUser);
```

Comment:
Due to explicit semantics behind the HTTP method names you don't need to have `userRouter.get('/find', getUserByName)`, here `userRouter.get('/', getUserByName)` will go

Reply:
the same for `POST`. By using POST you're saying that you want to create a resource

Reply:
Thanks Kate, I can see having those verbs can be redundant

Reply:
👍